modules/apps/batch-engine/batch-engine-test/src/testFunctional/tests/dependencies/blogPostingJsonT/blogPostingJsonT.json

```diff
@@ -14,9 +14,9 @@
 			"createStrategy": "INSERT",
 			"importStrategy": "ON_ERROR_FAIL",
 			"siteId": "20119",
-			"taskItemDelegateName": "DEFAULT",
 			"updateStrategy": "UPDATE"
 		},
+		"taskItemDelegateName": "DEFAULT",
```

Comment:
I think there are more cases like this in the whole liferay project

Reply:
I will check it